daemon/cmd/policy.go

```diff
@@ -688,22 +688,26 @@ func (h *putPolicy) Handle(params PutPolicyParams) middleware.Responder {
 
 	var rules policyAPI.Rules
 	if err := json.Unmarshal([]byte(params.Policy), &rules); err != nil {
-		metrics.PolicyImportErrorsTotal.Inc()
+		metrics.PolicyImportErrorsTotal.Inc() // To be deprecated in Cilium 1.14.
```

Comment:
As I understand, all of these should be instead:

```suggestion
		metrics.PolicyImportErrorsTotal.Inc() // Deprecated in Cilium 1.14, to be removed in 1.15.
```

Or did I get it wrong?

Reply:
You are right. Done. Thanks.